src/scanner/mod.rs

```diff
@@ -87,54 +98,72 @@ impl Scanner {
         open_sockets
     }
 
-    /// Given a port, scan it.
+    /// Given a port, scan it self.tries times.
     /// Turns the address into a SocketAddr
     /// Deals with the <result> type
     /// If it experiences error ErrorKind::Other then too many files are open and it Panics!
-    /// ese any other error, it returns the error in Result as a string
-    /// If no  errors occur, it returns the port number in Result to signify the port is open.
+    /// If any other error, it returns the error in Result as a string
+    /// If no errors occur, it returns the port number in Result to signify the port is open.
     /// This function mainly deals with the logic of Results handling.
     /// # Example
     ///
     ///     self.scan_port(10:u16)
```

Comment:
This does not look right, what's going on here?

Reply:
This comment is targetted only at the example given here

Reply:
I did not understand your comment.
But I corrected `ese` to `else` on my side and remove the double space.

Reply:
No no, your diff is fine, I meant the example that is here. It says `self.scan_port(10:u16)`, but that's not the method name and I don't know the `10:u16` syntax.

Reply:
I updated the documentation to follow the function name and argument on my local side.